### DISCUSSION TITLE
How to hide the folders/subfolders in the LORA view in Show Extra Networks

### DISCUSSION THREAD

**Discussion by astroprogs11** [anapnoe/stable-diffusion-webui-ux#47] [Q&A] [answered]
I want to hide the folders and only show the search bar and cards in the LORA tab in Extra Networks panel.

I'm guessing I need to edit ui_extra_networks_lora.py but I don't know what I should modify exactly.

```
import json
import os
import lora

from modules import shared, ui_extra_networks


class ExtraNetworksPageLora(ui_extra_networks.ExtraNetworksPage):
    def __init__(self):
        super().__init__('Lora')

    def refresh(self):
        lora.list_available_loras()

    def list_items(self):
        for name, lora_on_disk in lora.available_loras.items():
            path, ext = os.path.splitext(lora_on_disk.filename)
            yield {
                "name": name,
                "filename": path,
                "preview": self.find_preview(path),
                "description": self.find_description(path),
                "search_term": self.search_terms_from_path(lora_on_disk.filename),
                "prompt": json.dumps(f"<lora:{name}:") + " + opts.extra_networks_default_multiplier + " + json.dumps(">"),
                "local_preview": f"{path}.{shared.opts.samples_format}",
                "metadata": json.dumps(lora_on_disk.metadata, indent=4) if lora_on_disk.metadata else None,
            }

    def allowed_directories_for_previews(self):
        return [shared.cmd_opts.lora_dir]



```

**Comment by anapnoe** [comment, marked as answer]
you better not do it this way on your next update you will lose your changes a better solution is to add this line in user.css file
`[id$="2img_lora_subdirs"]{display:none;}`

**Reply by astroprogs11** [reply]
Thanks! Though I can't find user.css, where is it located?

EDIT: I duplicated style.css in the root folder, renamed it user.css and added the line and it worked as desired. I hope this was the correct way to go about this.

**Reply by anapnoe** [reply]
just create an empty file with the name user.css and add just the one line

**Comment by Psynbiotik** [comment]
Here is my user.css file which hides all folders!

user.css
[id$="2img_textual_inversion_subdirs"]{display:none;}
[id$="2img_hypernetworks_subdirs"]{display:none;}
[id$="2img_checkpoints_subdirs"]{display:none;}
[id$="2img_lora_subdirs"]{display:none;}
